src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPassBase.cs

```diff
@@ -368,7 +368,7 @@ bool IsImplicitStatementStart()
             if (owner.SpanStart == mappingSpan.Start &&
                 owner is CSharpStatementLiteralSyntax &&
                 owner.Parent is CSharpCodeBlockSyntax &&
-                owner.PreviousSpan() is CSharpTransitionSyntax)
+                owner.TryGetPreviousSibling(out var transition) && transition is CSharpTransitionSyntax)
```

Comment:
This wasn't flagged by PRISM, but removing all usages of `PreviousSpan` from the LSP editor just makes sense.